components/automate-ui/e2e/job-add.e2e-spec.ts

```diff
@@ -556,7 +556,7 @@ describe('Job Add', () => {
           scheduleToggle.click();
 
           expect(scheduleToggle.isSelected()).toEqual(true);
-          expect(startInputs.count()).toEqual(6);
+          expect(startInputs.count()).toEqual(5);
```

Comment:
We've removed the AM/PM input selection, so this test needed to be altered to look for 5 inputs instead of 6